<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HglMapperWindow</class>
 <widget class="QMainWindow" name="HglMapperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Sequence</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># pts</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Range</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Journal</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInstruments"/>
   <addaction name="actionDisplay"/>
   <addaction name="separator"/>
   <addaction name="actionStartSweep"/>
  </widget>
  <action name="actionInstruments">
   <property name="icon">
    <iconset>
     <normaloff>../resources/instruments.svg</normaloff>../resources/instruments.svg</iconset>
   </property>
   <property name="text">
    <string>Instruments</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="icon">
    <iconset>
     <normaloff>../resources/display.svg</normaloff>../resources/display.svg</iconset>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionStartSweep">
   <property name="icon">
    <iconset>
     <normaloff>../resources/play.svg</normaloff>../resources/play.svg</iconset>
   </property>
   <property name="text">
    <string>Start sweep</string>
   </property>
   <property name="toolTip">
    <string>Start sweep</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
